spec/latest/core/index.html

```diff
@@ -898,6 +898,10 @@ <h3>Description of HTTP Status Codes and Errors</h3>
       Hydra defines just one such subclass, namely the <i>Error</i> class.
       This provides an extensible framework to communicate error details to
       a client.</p>
+    
+    <p>Furthermore, a <i>Status</i> or <i>Error</i> returned by the server can also
+      be given an identifier. When dereferenced, the <i>Error</i> resource can provider
+      more detailed information or possible ways to resolve the problem, if applicable.<p/>
```

Comment:
How an `Error` can do that?
There is indeed an example of a `application/problem+json` with `invalid-params` property added on the [RFC 7807](https://tools.ietf.org/html/rfc7807), but it's not enlisted as a valid member, neither in that nor in hydra's spec.

Reply:
Not problem+json itself, but the linked resource. The RFC defines an `instance` property:

```json
{
    "type": "https://example.com/probs/out-of-credit",
    "instance": "/account/12345/msgs/abc"
}
```

That, when dereferenced, can include specific Hydra operations.